automation/run-local-utests.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+TOXWORKDIR="$HOME/.cache/.tox-nmstate"
+CONTAINER_IMAGE="nmstate/centos7-nmstate-utest"
+
+mkdir -p "$TOXWORKDIR"
+
+docker run \
+       --rm \
+       -ti \
+       -v $PWD:/workspace/nmstate:Z \
```

Comment:
It would be nicer to make `nmstate` a variable as well (maybe `${project}` and it could default to `${PWD##*/}`), so it can also be used for other projects.

Reply:
Nice, thanks. Done.